librz/core/cmd_api.c

```diff
@@ -330,6 +340,25 @@ RZ_API RzCmdDesc *rz_cmd_get_desc(RzCmd *cmd, const char *cmd_identifier) {
 	return res;
 }
 
+/**
+ * \brief Set the default mode of the command descriptor, if the type allows it.
+ *
+ * Command descriptors that support multiple output modes can also have a
+ * default one. This function can be used to set it.
+ */
+RZ_API void rz_cmd_desc_set_default_mode(RzCmdDesc *cd, RzOutputMode mode) {
```

Comment:
Add `RZ_NONNULL RzCmdDesc`

Reply:
I'm going to skip this because all rz_cmd.h api do not have any annotations. it is worth doing it in a separate PR in one shot.